modules/nextflow/src/main/groovy/nextflow/data/cid/CidStoreFactory.groovy

```diff
@@ -62,4 +62,11 @@ abstract class CidStoreFactory implements ExtensionPoint {
         }
     }
 
+    static void clean(){
```

Comment:
Let's mark it with `@TestOnly`, also `reset` may be a slightly better for consistency with other similar methods eg 

https://github.com/nextflow-io/nextflow/blob/138a1ff7e73089f516c44a11f7e72cf3a384ede7/modules/nextflow/src/main/groovy/nextflow/secret/SecretsLoader.groovy#L62-L64

Reply:
Changed